indico_toolkit/association/positioning.py

```diff
@@ -166,18 +185,11 @@ def get_horizontal_min_distance(pos1: dict, pos2: dict) -> float:
             raise ToolkitInputError(
                 "Predictions are not on the same page! Must enter a page height"
             )
-
+    
         min_distance_1 = abs(pos1["bbLeft"] - pos2["bbRight"])
```

Comment:
I'm not sure this logic makes sense... like imagine top box is 10left 20right and bottom box is 11left 19right- this isn't true representation of distance- I think this function assumes they're on the same line and can't overlap (not something we can assume)

Reply:
in the case where they overlap, should the distance be 0? If then, should we just add a conditional?

Reply:
I think this function intends to do something different, i.e. it assumes the objects are on the same line
otherwise it would have to min left -left , right - right too)- not sure it makes sense to change this function vs. rethink the way you're currently checking this